src/Auth/Environment.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace OpenBibIdApi\Auth;
+
+class Environment implements EnvironmentInterface
+{
+
+    /**
+     * The name of the environment.
+     *
+     * @var string
+     */
+    protected $name;
+
+    /**
+     * The base url.
+     *
+     * @var string
+     */
+    protected $baseUrl;
+
+    /**
+     * Creates a new \OpenBibIdApi\Auth\Environment.
+     *
+     * @param string $baseUrl
+     *   The base url.
+     * @param string name
+     *   The name of the environment (e.g. 'prod', 'qa', 'test').
+     */
+    public function __construct($baseUrl, $name = 'environment')
+    {
+        $this->baseUrl = $baseUrl;
+        $this->name = $name;
+    }
+
+    /**
+     * Get a production environment instance.
+     *
+     * @return \OpenBibIdApi\Auth\EnvironmentInterface
+     *   The production environment instance.
+     */
+    public static function production()
+    {
+        return new self('https://mijn.bibliotheek.be/openbibid/rest', 'prod');
```

Comment:
waarom static? Wanneer iemand overerft van Environment, willen we dan niet dat ze deze methode kunnen gebruiker zonder ze expliciet te moeten overschrijven?